### PR TITLE
Update Drawer.razor.cs 修改宽度可自适应

### PR DESCRIPTION
**Pull request by Nrtyck**
Fix Drawer width to string

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1390) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1390) before we can accept your contribution.<br/><br/>:x: Nrtyck [sign now](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1390)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/ant-design-blazor/ant-design-blazor?pullRequest=1390) it.</sub>

File: components/drawer/Drawer.razor.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ public partial class Drawer : AntDomComponentBase
 
         [Parameter] public string WrapClassName { get; set; }
 
-        [Parameter] public int Width { get; set; } = 256;
+        [Parameter] public string Width { get; set; } = "256";
 
         [Parameter] public int Height { get; set; } = 256;
```
Comment (ElderJames) [inline components/drawer/Drawer.razor.cs:78]: `Height` also need to be string.

**Review by ElderJames — CHANGES_REQUESTED** [review]

**Comment by ElderJames** [comment]
Thanks for contribution @Nrtyck 
Please sign the CLA, and fix the build errors.

File: site/AntDesign.Docs/Demos/Components/Drawer/demo/Multiple.razor [hunk 2]
```diff
@@ -25,7 +25,10 @@
        {
 
            this.visible2 = true;
-           wdFirstLayer += 260;
+
+           int w = int.Parse(wdFirstLayer) + 260;
```
Comment (ElderJames) [inline site/AntDesign.Docs/Demos/Components/Drawer/demo/Multiple.razor:29]: Please use `TryParse`

File: site/AntDesign.Docs/Demos/Components/Drawer/demo/Multiple.razor [hunk 3]
```diff
@@ -35,7 +38,12 @@
 
        void CloseDrawer()
        {
-           wdFirstLayer -= 260;
+           int w = int.Parse(wdFirstLayer);
```
Comment (ElderJames) [inline site/AntDesign.Docs/Demos/Components/Drawer/demo/Multiple.razor:41]: Please use `TryParse`

**Review by ElderJames — CHANGES_REQUESTED** [review]

**Comment by zxyao145** [comment]
Hi, @Nrtyck 
Please add the description for this PR.

**Comment by Nrtyck** [comment]
Drawer change  width and  height  vw vh

**Pull request closed without merging**